aspect/testing/rules/BUILD

```diff
@@ -2,6 +2,8 @@
 
 licenses(["notice"])  # Apache 2.0
 
+load("@rules_proto//proto:defs.bzl", "proto_library")
```

Comment:
We should probably move these into the global preamble if possible.

Reply:
Is this really necessary? I seem to able to still run everything without it.

Reply:
Right now, `rules_proto` is in the global preamble and automatically loaded if not defined, but the goal is to make these rules independent from Bazel releases, which isn't possible if they are in the preamble. Also, `rules_proto_{dependencies,toolchains}()` isn't called in the global preamble (and shouldn't be to avoid forcing users to download `rules_proto` if they don't use it).

Reply:
The incompatible flag has not been flipped yet, so everything works without these loads in Bazel 2.X. Once the flag is flipped (possibly in 3.0, https://github.com/bazelbuild/bazel/issues/8922), the loads are mandatory. See https://buildkite.com/bazel/bazelisk-plus-incompatible-flags/builds/404#6ec2b71a-4b6b-4e42-9c6d-0823aa81c642/1099-1101 which runs the CI with the flag flipped.

Reply:
> Right now, `rules_proto` is in the global preamble and automatically loaded if not defined

Sorry I meant the `tools/build_rules/prelude_bazel` file. I forgot the name of it when I wrote the comment, but it is prepended to every BUILD file, so that's why I called it the global preamble.

Reply:
Just tested it out with the flag. It seems to work fine. I'm gonna go with that instead of adding loads to every file.